apps/microtvm/reference-vm/base-box-tool.py

```diff
@@ -74,10 +74,31 @@ def parse_virtualbox_devices():
     return devices
 
 
+VIRTUALBOX_USB_DEVICE_RE = (
+    "UUID:.*([0-9a-z-]{36}).*\nVendorId:.*\(([0-9A-Z]{4})\).*\nProductId:.*\(([0-9A-Z]{4})\)\n"
+)
+
+
+def parse_virtualbox_attached_usb_devices(vm_uuid):
+    output = subprocess.check_output(["VBoxManage", "showvminfo", vm_uuid], encoding="utf-8")
```

Comment:
if you use `--machinereadable` here, it should be a bit easier to parse and more robust to updates in VBoxManage. would you be up for trying that?